dotnet/WPF/VisualLayerIntegration/VisualLayerIntegration/BarGraphUtility/BarGraph.cs

```diff
@@ -23,62 +23,59 @@
 //  ---------------------------------------------------------------------------------
 
 using SharpDX;
-using SharpDX.DirectWrite;
 using SharpDX.Direct2D1;
-using TextAntialiasMode = SharpDX.Direct2D1.TextAntialiasMode;
+using SharpDX.DirectWrite;
 using System;
 using System.Collections;
 using System.Collections.Generic;
-using SysWin = System.Windows;
 using Windows.UI;
 using Windows.UI.Composition;
+using SysWin = System.Windows;
 
 namespace BarGraphUtility
 {
     class BarGraph
     {
-        private float[] graphData;
+        private double[] _graphData;
```

Comment:
Now that you're doing initialization in the constructor, make as many of the fields readonly as you can.

Reply:
updated